scripts/endorser-connection.md

```diff
@@ -0,0 +1,36 @@
+To use the endorser agent in this project the following steps need to happen. Through swagger pages. 
+
+1. Generate invitation from endorser (POST `/connections/create_invitation`, receive invitation from tenant `/connections/recieve-invitation`
+
+1. Set author/tenant's connection to endorser with the following. POST to `/connections/{conn_id}/metadata`
+```
+{
+  "metadata": {
+   "endorser_info":{
+	   "endorser_did":"SVfHGCEEvEFmpBPcxgNqRR",
+	   "endorser_name":"endorser"	
+		},
+   "transaction_jobs": {
+      "transaction_their_job": "TRANSACTION_ENDORSER",
+      "transaction_my_job": "TRANSACTION_AUTHOR"
+    }
+	}
+}
+
+```
+
+3. Set endorsers connection to author/tenant with the following. POST to `/connections/{conn_id}/metadata`
+```
+{
+  "metadata":{
+    "transaction_jobs": {
+      "transaction_my_job": "TRANSACTION_ENDORSER",
+      "transaction_their_job": "TRANSACTION_AUTHOR"
+    }
+  }
+}
+```
+4. Out Of Band: Get tenant did/verkey from (with bearer token) GET `wallet/did` and POST that did/verkey to Endorser swagger at `/ledger/register-nym`. 
```

Comment:
Note - I'm working on an update to aca-py to add Endorser support for this (publishing author DID).